py/torch_tensorrt/dynamo/conversion/impl/activation/ops.py

```diff
@@ -0,0 +1,355 @@
+from typing import Any, Optional
+
+import numpy as np
+import tensorrt as trt
+import torch
+from torch.fx.node import Target
+from torch_tensorrt.dynamo._SourceIR import SourceIR
+from torch_tensorrt.fx.types import TRTNetwork, TRTTensor
+
+from .base import convert_activation
```

Comment:
This can be changed back to `from torch_tensorrt.dynamo.conversion.impl.activation.base import convert_activation`